content/src/content/jcr_root/apps/core/wcm/components/navigation/v2/navigation/README.md

```diff
@@ -0,0 +1,73 @@
+<!--
```

Comment:
Isn't this already part of https://github.com/adobe/aem-core-wcm-components/pull/1444 ?

Reply:
to simplify the management of this PR as some code used for the navigation component was used, I merged the navigation component's branch into this one.

Reply:
As I merged #1444 into development, I just updated this PR.

Reply:
LGTM